charts/kong/README.md

```diff
@@ -300,6 +300,12 @@ individual services: see values.yaml for their individual default values.
 * `manager`
 * `portal`
 * `portalapi`
+* `status`
+
+`status` is intended for internal use within the Kong pod only. Unlike other
```

Comment:
> internal use within the Kong pod only

I think you meant within the internal Kubernetes cluster only.

Reply:
status exposes health and metrics endpoints that are for in-cluster services only.